src/open_inwoner/openklant/services.py

```diff
@@ -786,6 +786,16 @@ def list_questions_for_zaak(self, zaak: Zaak, user: User) -> list[Question]:
             question: ContactMoment | None = getattr(ocm, "contactmoment", None)
             if question:
                 contactmomenten.append(question)
+
+        # filter contactmomenten via klantcontactmomenten
+        fetch_params = self.get_fetch_parameters(user)
+        klantcontactmomenten = self.fetch_klantcontactmomenten(**fetch_params)
+        cms_with_klantcontact = [kcm.contactmoment for kcm in klantcontactmomenten]
```

Comment:
I think verbose names help here:

```suggestion
        contactmomenten_within_klantcontacten = [kcm.contactmoment for kcm in klantcontactmomenten]
```